mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -313,7 +313,7 @@ def _read_protocol_line(self):
       # considering any blank output line to be EOF.
       line = line.strip()
       if (line.startswith('INSTRUMENTATION_RESULT:') or
-          line.startswith('SNIPPET ')):
+         line.startswith('SNIPPET ')):
```

Comment:
Fix according to Flake8 rule [E129](https://www.flake8rules.com/rules/E129.html)
> Visually indented line with same indent as next logical line